docker/Dockerfile

```diff
@@ -144,6 +135,17 @@ RUN cd /usr/local && \
 
 ENV KAHIP_ROOT=/usr/local/KaHIP
 
+# Download Install Gmsh SDK
```

Comment:
Can you use ubuntu's `python3-gmsh` and `gmsh`? https://packages.ubuntu.com/focal/python3-gmsh

Reply:
I don't think we want to - the Ubuntu gmsh package is already old, and after two years it will be really old.